src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -6,6 +6,7 @@ githubIntegrationURL: 'https://github.com/withastro/adapters/tree/main/packages/
 category: adapter
 i18nReady: true
 ---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 This adapter allows Astro to deploy your [`hybrid` or `server` rendered site](/en/core-concepts/rendering-modes/#on-demand-rendered) to [Netlify](https://www.netlify.com/).
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 17 to 17]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->> builder—its behavior out of the box—you

It looks strange without spaces before and after the dashes `—`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/withastro/docs/5978?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
This is why docs doesn't typically use emdashes... no spaces are technically correct in this case.